src/cephadm/cephadm

```diff
@@ -49,6 +49,8 @@ DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:master'
 DEFAULT_IMAGE_IS_MASTER = True
 DEFAULT_IMAGE_RELEASE = 'quincy'
 DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.18.1'
+DEFAULT_LOKI_IMAGE = 'docker.io/grafana/loki:2.4.0'
+DEFAULT_PROMTAIL_IMAGE = 'docker.io/grafana/promtail:2.4.0'
```

Comment:
we track the number of docker instances in the binary so the check needs to be updated

```
flake8 run-test: commands[0] | flake8 --config=tox.ini cephadm
flake8 run-test: commands[1] | bash -c 'test $(grep '"'"'docker.io'"'"' cephadm | wc -l) == 11'
ERROR: InvocationError for command /bin/bash -c 'test $(grep '"'"'docker.io'"'"' cephadm | wc -l) == 11' (exited with code 1)
```

Reply:
Thanks @adk3798 . Updated